lib/MusicBrainz/Server/Form/Artist.pm

```diff
@@ -69,21 +69,21 @@ sub validate {
     if ($self->field('type_id')->value &&
         $self->field('type_id')->value == 2) {
         if ($self->field('gender_id')->value) {
-            $self->field('gender_id')->add_error(l('Group artists cannot have a gender'));
```

Comment:
It sounds a bit odd to me without the article (I only ever see/hear gender identity as a 
countable noun).

Reply:
Ok, I reverted this change.